src/main/java/org/entando/kubernetes/validator/descriptor/PluginDescriptorValidator.java

```diff
@@ -216,6 +220,28 @@ private PluginDescriptor validateCustomIngressPathOrThrow(PluginDescriptor descr
         return descriptor;
     }
 
+    /**
+     * Validate the choice and assign a default, if needed.
+     *
+     * @param descriptor the plugin descriptor to validate
+     * @return InvalidBundleException if the value is not "canonical" or "custom" (case-insensitive)
+     */
+    private PluginDescriptor validateIngressHealthChoiceOrThrow(PluginDescriptor descriptor) {
+        if (StringUtils.isEmpty(descriptor.getHealthCheckIngress())) {
+            descriptor.setHealthCheckIngress(HEALTHCHECK_INGRESS_TYPE_DEFAULT);
+        } else {
```

Comment:
I think we could remove this `else` to simplify the code without breaking the flow, am I right?

Reply:
Correct, but if the first condition is met then performing validation test is not necessary; I'd leave it as is just for sake of efficiency,  but you just tell me :)